clients/admin-ui/src/features/common/table/v2/cells.tsx

```diff
@@ -115,7 +139,11 @@ export const IndeterminateCheckboxCell = ({
   dataTestId,
   ...rest
 }: CheckboxProps & { dataTestId?: string }) => (
-  <Flex alignItems="center" justifyContent="center">
+  <Flex
+    alignItems="center"
+    justifyContent="center"
+    onClick={(e) => e.stopPropagation()}
```

Comment:
@eastandwestwind The `stopPropagation()` call prevents the row's `onClick` from also being called when you try to toggle the checkbox.  This wasn't there before because the only place we had checkbox row selection was on the "add multiple systems" screen where rows don't have on-click behavior.